blocks/bricks/conv.py

```diff
@@ -494,16 +494,15 @@ class ConvolutionalSequence(Sequence, Initializable, Feedforward):
     def __init__(self, layers, num_channels, batch_size=None,
                  image_size=(None, None), border_mode=None, tied_biases=None,
                  **kwargs):
-        self.layers = layers
+        self.layers = [a if isinstance(a, Brick) else a.brick for a in layers]
```

Comment:
See #1163. If `application_methods` gets modified you want that change to be reflected in `layers`. Having the same information in two places is asking for one of them to become stale (we already sort of have that, in that we have to mirror changes to `application_methods` with possible changes to `children`, which is what #1163 addresses in order to make Sequence surgery less of a pain; having to also manage `layers` for `ConvolutionalSequence` is yet another layer).